ES/ESum042.xml

```diff
@@ -30,608 +28,321 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>Ethio-SPaRe</collection>
                   <idno>UM-042</idno>
```

Comment:
no images are present on the server (only the first User folder up to 35 was uploaded) - should they be?

Reply:
Good question, but not for the moment.